app/blueprints/fund_builder/templates/build_application.html

```diff
@@ -35,8 +35,14 @@ <h2 class="govuk-heading-m">{{ fund.short_name }} - {{round.title_json["en"]}} (
                     <h3 class="govuk-heading-m">{{ section.index }}. {{ section.name_in_apply_json["en"] }}</h3>
                 </span>
                 <span class="app-task-list__task-actions">
-                    <a href='{{ url_for("build_fund_bp.section", round_id=round.round_id, section_id=section.section_id) }}'>Edit</a>
-                    <a href='{{ url_for("build_fund_bp.section",round_id=round.round_id, section_id=section.section_id, action="remove") }}'>Remove</a>
+                    <a class="govuk-link--no-visited-state" href='{{ url_for("build_fund_bp.section", round_id=round.round_id, section_id=section.section_id) }}'>Edit</a>&nbsp;
```

Comment:
To give space cant we use CSS ? may be

Reply:
Probably yes in future. But we don't have proper designs for these screens yet, they are just POC, so when we have proper designs we can spend the time on better CSS 👍